tests/python/gaia-ui-tests/gaiatest/tests/functional/lockscreen/test_lockscreen_unlock_to_camera_with_passcode.py

```diff
@@ -2,6 +2,7 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
+from gaiatest import GaiaTestCase
```

Comment:
You're right it's used actually. As the testcase depends on `PasscodeTestCase`, we need to use `PasscodeTestCase.setUp()` instead.